app/templates/tools/includes/list.html

```diff
@@ -37,7 +37,17 @@ <h2 class="heading-medium">Your tools</h2>
         </td>
       </tr>
       {% endfor %}
-    {% else %}
+    {% endif %}
+
+    {% set showRstudio = true %}
+    {% for tool in tools %}
+      {% if tool.name == 'rstudio' %}
```

Comment:
As discussed offline, this probably needs to be `tool.app_label` but the rest looks good to me 👍

Reply:
Oh, and I also think you could avoid this loop and use the one above (but don't worry about that!)